### DISCUSSION TITLE
Import saved html file to freshly installed Zim, Linux mint 22

### DISCUSSION THREAD

**Discussion by Condobloke** [zim-desktop-wiki/zim-desktop-wiki#2677] [Q&A] [answered]
Linux Mint 22 installled...previously Linux Mint 21.3.....Saved the entire Zim as "FINAL SEPT 10 2024 INSTALL_Linux_Mint.html"

problem: i "imported" that file to the current ZIM install. it does NOT occupy the Index space as it should.

Where have I gone wrong?

Is there a procedure to put it where it belongs?

(perhaps naively, I thouht that seeing the Zim app saves as a html it would also restore that file fairly simply)

**Comment by jaap-karssenberg** [comment, marked as answer]
Zim operates on plain text files. To move a notebook to another system,
just move the folder with text files

The html export is useful to e.g. present zim content in a browser or
upload to a website.

Currently Zim does not support html import although it is on the wishlist.

Regards,

Jaap

Op za 14 sep. 2024 01:36 schreef Condobloke ***@***.***>:
